DashKitUI/Views/DeactivatePodView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  DeactivatePodView.swift
+//  DashKitUI
+//
+//  Created by Pete Schwamb on 3/9/20.
+//  Copyright © 2020 Tidepool. All rights reserved.
+//
+
+import SwiftUI
+import LoopKitUI
+
+struct DeactivatePodView: View {
+    
+    @ObservedObject var viewModel: DeactivatePodViewModel
+
+    @Environment(\.verticalSizeClass) var verticalSizeClass
+
+    var body: some View {
+        GuidePage(content: {
+            VStack {
+                LeadingImage("Pod")
+
+                HStack {
+                    Text("Please deactivate the pod. When deactivation is complete, remove pod from body.")
+                        .fixedSize(horizontal: false, vertical: true)
+                    Spacer()
+                }
+            }
+        }) {
+            if self.viewModel.state.showProgressDetail {
+                VStack {
+                    self.viewModel.error.map {ErrorView($0).accessibility(sortPriority: 0)}
```

Comment:
[nit] I still am not a fan of this syntax to show the error if it isn't nil.  Maybe something like this could be used?
```swift

extension Optional {
    
    func ifNotNil<U>(_ doIfTrue: (Wrapped) throws -> U) rethrows -> U? {
        return try self.map(doIfTrue)
    }

}
```

Then this could look like:

```swift
self.viewModel.error.ifNotNil {ErrorView($0).accessibility(sortPriority: 0)}
```

...just an idea.

Reply:
I believe map for optionals is becoming idiomatic, so I think the custom `ifNotNil` might present a more obscure reading challenge.